test/runtime/adapter-read-timeseries.spec.js

```diff
@@ -18,6 +18,19 @@ describe('read testTimeseries', function () {
         });
     });
 
+    it('requires -from and -to to be different', function() {
+        return check_juttle({
+            program: 'read testTimeseries -from :2014-01-01: -to :2014-01-01:'
+        })
+        .then(function(result) {
+            throw new Error('unexpected success');
+        })
+        .catch(function(err) {
+            expect(err.code).equal('RT-FROM-TO-EQUAL-ERROR');
+            expect(err.message).equal('read testTimeseries -from/-to must not be equal');
```

Comment:
You could only test for the code, both is probably a bit too strict/redundant, but OK.